tests/b2share_unit_tests/records/test_records_serializers.py

```diff
@@ -58,7 +58,7 @@ def test_records_serializers_dc(app, test_records_data):
     with app.app_context():
         pid, record = make_record(test_records_data)
         rec = {
-            '_source': RecordIndexer._prepare_record(
+            '_source': RecordIndexer()._prepare_record(
```

Comment:
_prepare_record can not be invoked without having the parenthases with the RecordIndexer class